.github/workflows/build-binaries.yml

```diff
@@ -216,7 +120,7 @@ jobs:
           cd bin
           for idir in */;
           do
-            (cd "$idir"; zip -r "../${idir%/}.zip" .);
+            (cd "$idir"; rm dungeon-revealer || true; zip -r "../${idir%/}.zip" .);
```

Comment:
why tying to remove dungeon-revealer?

Reply:
That removes the script that lives in bin. It gets added to the artifact because of the wildcard on line 42.